src/Terminus.php

```diff
@@ -296,6 +296,7 @@ private function addBuiltInCommandsAndHooks()
     {
         // List of all hooks and commands. Update via 'composer update-class-lists'
         $this->commands = [
+            'Consolidation\\Filter\\Hooks\\FilterHooks',
```

Comment:
Good find; wonder when this got dropped.

Reply:
That didn't... those are more recent changes already on `3.x` that I merged back.  I closed this PR because I merged the 3.x changes back to Jordan's branch too but they were still showing in this PR, and they're not real changes, so I made a new PR that doesn't show them.